language/stdlib/modules/libra_account.mvir

```diff
@@ -275,9 +275,8 @@ module LibraAccount {
         return;
     }
 
-    // Creates a new account at `fresh_address` with the `initial_balance` deducted from the
-    // transaction sender's account
-    public create_new_account(fresh_address: address, initial_balance: u64) acquires T {
```

Comment:
This was a relic from the past, where we were nervous about creating accounts with 0 balance. The field was here so we could easily update the min-balance in the future. It seems like we are okay with that for now?

Reply:
Good question. My thought is that because the newly created account can always send the money back to the creator, a minimum balance doesn't buy us any sort of protection over irresponsible account creation. Charging a high gas cost for account creation seems like a more reliable way to ensure responsible creation behavior.